js/app.js

```diff
@@ -33,7 +35,7 @@ var Polyglot = require('node-polyglot'),
     loadProfileCountdownInterval,
     loadProfileCountdown = 5;
 
-serverUrlLocal = localStorage.getItem("serverUrl") || "http://localhost:18469/api/v1/";
+serverUrlLocal = "http://192.168.1.22:18469/api/v1/" || localStorage.getItem("serverUrl") || "http://localhost:18469/api/v1/";
```

Comment:
More importantly, why is it overriding the local settings? As written, the first check in this line will always be true, the second and third checks will never happen.